app/views/users/_follow.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= form_for current_user.active_relationships.build, remote: true do |f| %>
```

Comment:
I have found that, you defined helper method
```ruby 
def build_active_relationships
  current_user.active_relationships.build
end
```
Why you didn't used it!!

Reply:
I try use it, but it exception when rails s